### PR TITLE
Add directories required, dockerfiles, and update the readme

### PR DESCRIPTION
**Pull request by halfwit**
This PR is not quite ready yet, but this serves well for documenting progress.

### CODE REVIEW

**Comment by halfwit** [comment]
**TODO:**
- The MYSQL environment is currently set via named files in /var/secrets, but presently there is nothing being populated.

- nginx only runs on port 
  - Need to document certificate creation, and update the dockerfiles to put them where they need to go

- the mysql conf will need some updates, cc @itsrachelfish

**Comment by halfwit** [comment]
Hey @itsrachelfish, would be probably sane to squash all this on the merge

**Comment by halfwit** [comment]
Putting back into draft until this is thoroughly tested with the Let's Encrypt bits added in.

**Comment by halfwit** [comment]
As is, this should work perfectly. I'd love to have a few people test run this, before merging.

File: .env [hunk 1]
```diff
@@ -1,8 +1,9 @@
 APP_ENV=local
```
Comment (itsrachelfish) [inline .env:1]: The .env file is supposed to be gitignored

**Review by itsrachelfish — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    container_name: swag
+    restart: unless-stopped
+    environment:
+      PUID: 1000
+      PGID: 1000
+      TZ: $TIMEZONE
+      URL: $SITE_URL
+      SUBDOMAINS: $SUBDOMAINS
+      VALIDATION: http
+      ONLY_SUBDOMAINS: "true"
```
Comment (itsrachelfish) [inline docker-compose.yml:53]: Does this mean using the volunteer database without a subdomain is not supported?

**Comment by halfwit** [comment]
@itsrachelfish Ready for final review.

**Review by itsrachelfish — APPROVED** [review]
Wow, it really has never been easier to set up this project!

**Pull request merged**